infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationUtils.kt

```diff
@@ -83,31 +83,33 @@ fun asCsvFile(items: List<PublicationTableItem>, timeZone: ZoneId, translation:
 private fun enumTranslationKey(enumName: LocalizationKey, value: String) = "enum.${enumName}.${value}"
 
 private fun <T> formatChangeValue(translation: Translation, value: ChangeValue<T>): String {
-    val oldValue = if (value.localizationKey != null && value.oldValue != null) translation.t(
-        enumTranslationKey(
-            value.localizationKey, value.oldValue.toString()
-        )
-    ) else if (value.oldValue == null) null else value.oldValue.toString()
-    val newValue = if (value.localizationKey != null && value.newValue != null) translation.t(
-        enumTranslationKey(
-            value.localizationKey, value.newValue.toString()
-        )
-    ) else if (value.newValue == null) null else value.newValue.toString()
 
-    return "${if (oldValue != null) oldValue else ""} -> ${if (newValue != null) newValue else ""}"
+    return "${
+        (if (value.localizationKey != null && value.oldValue != null) translation.t(
```

Comment:
Nämä iffittelyt voisi varmaan funktioida. 2 osaa näyttää lähes identtisiltä ja noin pitkä koodi string-templatessa on vähän erikoinen.

Reply:
IDEAn mielestä tää oli se way to go 😅